test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5106,7 +5106,7 @@ def revert_to_defaults(self, reverthooks=True, revertqueues=True,
             hooks = self.status(HOOK, level=logging.DEBUG)
             hooks = [h['id'] for h in hooks]
             if len(hooks) > 0:
-                self.manager(MGR_CMD_DELETE, HOOK, id=hooks, expect=True)
+                self.manager(MGR_CMD_DELETE, HOOK, id=hooks)
```

Comment:
I think all of the places where the framework was previously doing an expect=True, we should now do an expect() explicitly to check the same. This will help catch errors at the setUp() stage itself.

Reply:
As I said in @hirenvadalia's comment, I don't think we should.  The logic behind removing expect=True was it was not necessary.  Adding in expect() calls will just make revert_to_defaults() take longer for no real benefit.

Reply:
Ok, but by not doing an expect, I fear that there could be race conditions: setUp fires off PBS commands which might still be running (on a slow machine for example) while setUp ends and the test starts.

Reply:
I'm not sure I understand.  The manager() call doesn't return until qmgr has been called.  This means the value has been set.  How can we have a race condition on slow machines?

Reply:
Ah, yes, you are correct, manager() is a blocking call so it shouldn't cause any issues, my bad.